relay-profiling/src/measurements.rs

```diff
@@ -20,12 +20,17 @@ pub struct MeasurementValue {
 }
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
-#[serde(rename_all = "lowercase")]
+#[serde(rename_all = "snake_case")]
 pub enum MeasurementUnit {
     #[serde(alias = "ns")]
     Nanosecond,
     #[serde(alias = "hz")]
     Hertz,
+    Byte,
+    Bool,
+    MemoryPressureEnum,
+    ThermalStateEnum,
+    Percent,
```

Comment:
If we ever start deserializing profiles in customer relays, we will have to add a fallback variant here (e.g. `Unknown(String)`) to achieve forward compatibility. See https://github.com/getsentry/relay/pull/1464.